qulice-maven-plugin/src/test/java/com/qulice/maven/DefaultMavenEnvironmentTest.java

```diff
@@ -128,4 +128,30 @@ public final void passPathsWithWhitespaces()  throws Exception {
             Matchers.notNullValue()
         );
     }
+
+    /**
+     * DefaultMavenEnvironment can produce empty collection when no matches
+     * with checker.
+     * @throws Exception If something wrong happens inside
+     * @todo #676:30min Ideally integration test required to reproduce NPE.
+     *  It should contains at least one exclude rule in
+     *  configuration.excludes block and dependency (I got it
+     *  locally with org.hibernate:hibernate-annotations:3.5.6-Final.
+     *  I suppose, dependency must be compile-time scope but it is only my
+     *  suggestions as far as junit with hamcrest didn't work.
```

Comment:
@mkordas Ah, I understand your misunderstanding. )) 
The behavior of dependency validator is not clear for me. If to add junit dependency to local project (I told that reproduced locally), but no effect. Only hibernate dependency affected, so I made suggestion that only dependencies with compile scope affects. Maybe I should delete this, because it is only suggestion and can confuse future developer